boards/arm/nucleo_wb55rg/nucleo_wb55rg.yaml

```diff
@@ -6,7 +6,7 @@ toolchain:
   - zephyr
   - gnuarmemb
   - xtools
-ram: 96
+ram: 256
```

Comment:
This value is not used for documentation, but to inform twister about maximum available RAM size to avoid tests requesting more RAM than available on targets. So it has to be coherent with what is given in device tree (192 based on the dts change).

Reply:
Alright, fixed!